src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/LdapFactoryTrait.php

```diff
@@ -27,6 +27,11 @@
  */
 trait LdapFactoryTrait
 {
+    public function getKey(): string
```

Comment:
The `string` PHP return type is missing from all other factories. If I understand correctly, I should make sure the debug class loader can fix this automatically and create a PR using the loader, correct? @nicolas-grekas